examples/example_notebooks/keras_rl2_example.ipynb

```diff
@@ -0,0 +1,1483 @@
+   "metadata": {},
+   "source": [
+    "# Training an RL agent with Keras-RL2 using a GEM environment\n",
+    "\n",
+    "This notebook serves as an educational introduction to the usage of Keras-RL2 using a GEM environment. Goal of this notebook is to give an understanding of what Keras-RL2 is and how to use it to train and evaluate a Reinforcement Learning agent which is able to solve a current control problem within the GEM toolbox."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Installation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Before you can start you need to make sure that you have installed both, gym-electric-motor and Keras-rl2. You can install both easily using pip:\n",
+    "\n",
+    "- ```pip install gym-electric-motor```\n",
+    "- ```pip install keras-rl2```\n",
+    "\n",
+    "Alternatively, you can install them and their latest developer version directly from GitHub (recommended) :\n",
+    "\n",
+    "- https://github.com/upb-lea/gym-electric-motor\n",
+    "- https://github.com/wau/keras-rl2\n",
+    "\n",
+    "For this notebook, the following cell will do the job:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install -q git+https://github.com/upb-lea/gym-electric-motor.git git+https://github.com/wau/keras-rl2.git"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. Setting up a GEM evironment\n",
+    "\n",
+    "The basic idea behind reinforcement learning is to create a so-called agent, that should learn by itself to solve a specified task in a given environment. \n",
+    "This environment gives the agent feedback on its actions and reinforces the targeted behavior.\n",
+    "In this notebook, the task is to train a controller for the current control of a *Permanent Magnet Synchronous Motor* (*PMSM*).\n",
+    " \n",
+    "In the following, the used GEM-environment is briefly presented, but this notebook does not focus directly on the detailed usage of GEM. If you are new to the used environment and interested in finding out what it does and how to use it, you should take a look at the [GEM cookbook](https://colab.research.google.com/github/upb-lea/gym-electric-motor/blob/master/examples/example_notebooks/GEM_cookbook.ipynb).\n",
+    "\n",
+    "To save some space in this notebook, there is a function defined in an external python file called **getting_environment.py**. If you want to know how the environment's parameters are defined you can take a look at that file. By simply calling the **get_env()** function from the external file, you can set up an environment for a *PMSM* with discrete inputs.\n",
+    "\n",
+    "The basic idea of the control setup from the GEM-environment is displayed in the following figure. \n",
+    "\n",
+    "![](../../docs/plots/SCML_Overview.svg)\n",
```

Comment:
The picture is somehow distorted -> the headlines "controller" and "environment" are displaced